ross/results.py

```diff
@@ -637,6 +638,30 @@ def plot(self, dof, ax0=None, ax1=None, **kwargs):
 
 class ModeShapeResults(Results):
     def plot(self, mode=None, evec=None, fig=None, ax=None):
```

Comment:
When mode value is set to default, the code is returning an Error, as you can see in:

![image](https://user-images.githubusercontent.com/32821252/59239881-78a8c000-8bd9-11e9-8b57-7340c51f0d12.png)
I think we can remove this variable default value or change it to 0, what do you think?

Reply:
This error was obtained by running the following code:
```
import ross as rs 
mode = rs.rotor_example().run_mode_shapes()
```

Reply:
I have changed mode to be a required positional argument.
If you try: 
```python
import ross as rs 
mode = rs.rotor_example().run_mode_shapes()
mode.plot()
```
it will raise with an error message that the user can understand:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-5fd538f603a5> in <module>
----> 1 mode.plot()

TypeError: plot() missing 1 required positional argument: 'mode'
```